src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -77,6 +77,7 @@ def _prepare_home_dir() -> Path:
 PF_SERVICE_LOG_FILE = "pfs.log"
 PF_SERVICE_HOUR_TIMEOUT = 1
 PF_SERVICE_MONITOR_SECOND = 60
+PF_SERVICE_WAITRESS_THREADS = 16
```

Comment:
maybe make this not mention thread, waitress is based on thread, but other WSGI host like gunicorn is process based.
Does this sounds better?
PF_SERVICE_WORKER_COUNT

Reply:
yes, maybe worker here is better

Reply:
updated to `PF_SERVICE_WORKER_NUM`